### ISSUE TITLE
Next: header megamenu accessibility

### ISSUE THREAD

**Issue by Fupete** [italia/designers.italia.it#499]
a11y ⚠️ to fix: 
- main menu items must be `button`s and not `a`s, as they trigger the action to open megamenu (as in Bootstrap Italia megamenu example) and they are not links. 
  - FYI 🔴 this is an issue also on Bootstrap Italia documentation of the [header](https://italia.github.io/bootstrap-italia/docs/menu-di-navigazione/header/) @astagi 
- the first item on the megamenu is better rendered like a simple `a`, without the `ul` and `li` structure for just one item

To be fixed in #474  

cc ⚠️ 💙 @astagi we have to fix this also in Bootstrap Italia header nav documentation too, where the dropdowns and the megamenu are links and not the right buttons... 

Reviewed with Fabrizio. 
_Originally posted by @Fupete in https://github.com/italia/designers.italia.it/issues/474#issuecomment-1331900941_

**Comment by gpeirolo** [comment]
@Fupete fixed in #502, please review it.

**Comment by Fupete** [comment]
Just ccommented there on the PR: @gpeirolo maybe https://github.com/italia/designers.italia.it/pull/474 ⚠️ one of the menus divided into three should have been merged first? 🤔
